syntax = "proto3";

package messages;

option go_package = "proto/messages";

service Server {
    rpc GetFortune(GetFortuneRequest) returns (GetFortuneResponse) {}
}

message GetFortuneRequest {
    bool offensive = 1;
}

message GetFortuneResponse {
    string fortune = 1;
}
